src/plone/restapi/tests/http-examples/linkintegrity_get.resp

```diff
@@ -7,9 +7,18 @@ Content-Type: application/json
         "@type": "Document",
         "breaches": [
             {
-                "@id": "http://localhost:55001/plone/doc-1",
-                "title": "First document",
-                "uid": "SomeUUID000000000000000000000001"
+                "sources": [
+                    {
+                        "@id": "http://localhost:55001/plone/doc-1",
+                        "title": "First document",
+                        "uid": "SomeUUID000000000000000000000001"
+                    }
+                ],
+                "target": {
+                    "@id": "http://localhost:55001/plone/doc-2",
+                    "title": "Second document",
+                    "uid": "SomeUUID000000000000000000000002"
+                }
```

Comment:
@jaroel @tisto @sneridagh  I don't understand why it's necessary to change the shape of the response like this. The new `target` inside `breaches` is just duplicating information from the top level list (lines 6-7), isn't it? (Of course, it's simple to add the title and uid at the top level if they are needed.)

Maybe I am missing something?

Reply:
@davisagli The target is different for (also deleted) subitems.

https://github.com/plone/plone.restapi/blob/7baab0eb68207dd9abc3816e333e04eb9cdcd000/src/plone/restapi/tests/test_services_linkintegrity.py#L349-L352 gives an idea of that. Does this help?

Reply:
@jaroel Okay, that helps. But do the breaches for a subitem need to be listed under their parent? If not, we could keep the same shape (with all the targets at the top level for both items the user selected and subitems, and each one contains a list of its breaches/sources).

Just trying to avoid breaking compatibility if there isn't an important reason.

Reply:
@davisagli yep; I need to push it one level up and it should work as-is. 🥇

Reply:
@davisagli could have another look?